server-spi-private/src/main/java/org/keycloak/forms/login/LoginFormsPages.java

```diff
@@ -24,7 +24,7 @@
 
     LOGIN, LOGIN_USERNAME, LOGIN_PASSWORD, LOGIN_TOTP, LOGIN_CONFIG_TOTP, LOGIN_WEBAUTHN, LOGIN_VERIFY_EMAIL,
     LOGIN_IDP_LINK_CONFIRM, LOGIN_IDP_LINK_EMAIL,
-    OAUTH_GRANT, LOGIN_RESET_PASSWORD, LOGIN_UPDATE_PASSWORD, LOGIN_SELECT_AUTHENTICATOR, REGISTER, INFO, ERROR, ERROR_WEBAUTHN, LOGIN_UPDATE_PROFILE,
+    OAUTH_GRANT, LOGIN_RESET_PASSWORD, LOGIN_UPDATE_PASSWORD, LOGIN_SELECT_AUTHENTICATOR, REGISTER, REGISTER_DYNAMIC, INFO, ERROR, ERROR_WEBAUTHN, LOGIN_UPDATE_PROFILE,
```

Comment:
Would be better `REGISTER_USER_PROFILE`. Perhaps easier to map with the feature.